src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -116,6 +115,17 @@ public ResponseEntity<List<PreContingencyLimitViolationResultDTO>> getNResult(@P
                 : ResponseEntity.notFound().build();
     }
 
+    @PostMapping(value = "/results/{resultUuid}/n-result/csv", produces = APPLICATION_OCTET_STREAM_VALUE)
```

Comment:
I think it's preferable this way since we already have different endpoint for paged results, it makes the endpoints more REST-friendly
But we can ask a third person their opinion if you want